### ISSUE TITLE
Code not found

### ISSUE THREAD

**Issue by laobuda** [Apress/practical-ddd-in-enterprise-java#6]
The code snippet in the book is no where to be found in this project for instance on page 75 in the "Commands" section listing 13-5 mentions a method specifyNewRoute that I cannot find anywhere and this just one of the examples that I ran to. I am already dissapointed to have bought this book.

**Comment by armzim** [comment]
Yeah, there's missing code, quite a few bugs, you name it. Do yourself a favour and get the code from https://github.com/practicalddd/practicaldddbook/tree/master/Chapter5 ;)
